terraform/stacks/ecr/stack.tf

```diff
@@ -17,6 +17,7 @@ variable "repositories" {
     "cron-resource",
     "csb",
     "email-resource",
+    "external-domain-broker",
```

Comment:
```suggestion
    "external-domain-broker-testing",
```

Reply:
This image is only used for testing in our pipeline, so it might be helpful if the name clarifies that